### ISSUE TITLE
Pascal VOC to VOTT 

### ISSUE THREAD

**Issue by vJenny** [Microsoft/VoTT#292]
It would be great to have a functionality to import the Pascal VOC format to the internal VOTT format to review and reimport existing markup.

**Comment by athlete1** [comment]
This would be a wonderful feature.  Huge time saver.  Currently we use pascal voc format for all of our training and we have the ability for our model to run through all of our images and save the image to folders along with the pascal voc xml files but currently don't have a way to load those folders back into VOTT to review or correct tags and reexport.

**Comment by Chappie74** [comment]
Indeed. I wanted this too. About two years later still not implemented. So I created a script to do it for me. Check out the repo https://github.com/Chappie74/convert_pascalvoc_to_vott
